src/components/Plan.js

```diff
@@ -18,80 +18,114 @@ import React, { Component } from 'react';
 
 import TokenInput from './TokenInput';
 
-class Plan extends Component {
+class IssueTree extends Component {
 
     render() {
-        let phases = [...new Set(
-            [].concat(...this.props.issues.map(issue =>
-                issue.labels.map(label => label.name)
-            )))]
-            .filter(label => label.startsWith("phase:"))
-            .sort((a, b) => {
-                return Number(a.split(":")[1]) -
-                    Number(b.split(":")[1]);
-            });
+        let fields = this.props.fields;
+        let items = this.props.items;
+        let renderItem = this.props.renderItem;
 
-        let phasedIssues = {};
-        phases.forEach(phase => {
-            phasedIssues[phase] = this.props.issues.filter(issue => {
-                return issue.labels.map(label => label.name).includes(phase);
-            });
-        });
+        if (fields.length === 0) {
+            return items.sort(this.props.sortItems).map(item => renderItem(item));
+        }
+
+        let thisLevel = fields[0];
```

Comment:
Could do `let { field, sort } = fields[0]` and use destructuring to pull out the field properties, if you like.

Reply:
Much nicer 👍